test/TEST-35-LOGIN/test.sh

```diff
@@ -9,6 +9,7 @@ TEST_DESCRIPTION="Tests for systemd-logind"
 
 test_append_files() {
     image_install -o evemu-device evemu-event crond crontab
+    image_install "${BUILD_DIR}"/test-session-properties
```

Comment:
@mrc0mmand Is this necessary? I thought our test binaries are always installed into the basic image.

Reply:
I think it shouldn't be necessary, but the path in the test itself needs to be adjusted (to `/usr/lib/systemd/tests/test-session-properties` IIRC).

Reply:
Ah, nevermind, the test already uses the path, so this can be simply dropped altogether.

Reply:
Good, I'll drop it.